### DISCUSSION TITLE
Does nginder support IPv6？

### DISCUSSION THREAD

**Discussion by zhougy1989** [naver/ngrinder#753] [Q&A]
Does nginder support IPv6？Does it have anything to do with the server? The domain name resolution error always occurs during the test

**Comment by junoyoon** [comment]
Okay. Could you post the error log?

**Reply by zhougy1989** [reply]
java.net.UnknownHostException: www.xxxx.cn  ,
but, on the Linux server, I have pointed the domain name to the IPv6 address in the hosts file  
![image](https://user-images.githubusercontent.com/50607317/106405182-4b74e980-6470-11eb-9dc8-c66d4c64641d.png)

**Reply by zhougy1989** [reply]
When I send a get request, do I want to request the IPv6 address directly or can I request the domain name？

**Reply by junoyoon** [reply]
@zhougy1989 
OK. I see.. However, I think it will take long time to fix it becaz currently we don't have any IP6 machine. 
Or.. you're welcomed to make a contribution.

**Reply by zhougy1989** [reply]
sorry,it is the company‘s resources，i don't have authority
and  I installed ngrinder in docker. Will this affect the domain name in hosts？
